### ISSUE TITLE
Mobile: button and tags styling issues

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#870]
![Image](https://user-images.githubusercontent.com/10619898/185463280-e21fd4f2-0088-462a-89a3-9f303f693219.png)

Android 12/ talk back/ Chrome

The following buttons and labels have styling problems: 
- [ ] Save next button on individual page creation
- [ ] Add to my notification on the organization page view
- [ ] label for people with disabilities, supporters and Deaf people on the individual page view
- [ ] Label for information and communication technologies on the individual page view areas of inetrest

**Comment by cherylhjli** [comment]
@sepidehshahi do you mean all 4 items on your list have the styling issue in the screenshot?

**Comment by sepidehshahi** [comment]
Yes @cherylhjli I only had a screenshot of one item, but all 4 items have styling issues.

**Comment by cherylhjli** [comment]
@greatislander Please have the text wrap to the next line for both tags and buttons: 
<img width="291" alt="Screen Shot 2022-08-23 at 3 28 07 PM" src="https://user-images.githubusercontent.com/9353905/186249146-89413e4e-aedb-43cc-9e87-d367fd7b2ef5.png">
<img width="468" alt="Screen Shot 2022-08-23 at 3 28 05 PM" src="https://user-images.githubusercontent.com/9353905/186249153-f759c626-7a2c-4009-a967-786aaabe53e9.png">

https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1134%3A11567

**Comment by greatislander** [comment]
@cherylhjli if button text is too long for the screen and has to wrap, should the button have a maximum width? I expect it should be left-aligned and narrower than the column width (unless on mobile).

**Comment by cherylhjli** [comment]
yes the button being left aligned and narrower than the column width sounds good to me!
